### PR TITLE
874256-UG documentation added for .NET MAUI Theme support

### PR DESCRIPTION
**Pull request by AarthiArjunan**
### **Description**

- Added UG content for the .NET MAUI theme support

#### **Added topics**

- Applying themes

- How to

### CODE REVIEW

File: MAUI/Themes/Themes.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+Syncfusion themes allow you apply colors across all the Syncfusion controls with a uniform approach and provide consistent look and feel to your applications. This section covers the following topics: 
+
+* Applying light and dark themes
+* Overriding the default theme
+* Creating your own theme
+
+## Applying light and dark themes
+
+By default, Syncfusion offers support for both light and dark themes through the inclusion of a Syncfusion Theme dictionary. As names suggest, these themes will have colors with light and dark color contrasts, respectively.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Themes/Themes.md:20]: Syncfusion Theme dictionary - this is a class name. So highlight it and add it properly.

Comment (AarthiArjunan) [reply MAUI/Themes/Themes.md]: Changes addressed.

File: MAUI/Themes/HowTo.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+layout: post
+title: How to customize theming in Syncfusion controls
+description: Learn here about how to switch between light and dark themes in Syncfusion Maui controls and more details.
+platform: maui
+control: General
+documentation: ug
+---
+
+# Switching between light theme and dark theme
+
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Themes/HowTo.md:11]: remove the unwanted empty lines

File: MAUI/Themes/HowTo.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+{% highlight C# %} 
+void UpdateTheme(object sender, System.EventArgs e)
+{
+    ICollection<ResourceDictionary> mergedDictionaries = Application.Current.Resources.MergedDictionaries;
+    if (mergedDictionaries != null)
+    {
+        var theme = mergedDictionaries.OfType<SyncfusionThemeResourceDictionary>().FirstOrDefault();
+        if (theme != null)
+        {
+            if (theme.SfVisualTheme is SfVisuals.DarkDefault)
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Themes/HowTo.md:24]: change the property name, theme.SfVisualTheme is **SfVisuals.MaterialDark**;

File: MAUI/Themes/Themes.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+* Applying light and dark themes
+* Overriding the default theme
+* Creating your own theme
+
+## Applying light and dark themes
+
+By default, Syncfusion offers support for both light and dark themes through the inclusion of a `SyncfusionThemeDictionary`. As names suggest, these themes will have colors with light and dark color contrasts, respectively.
+
+To apply themes to your application, merge the following items:
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Themes/Themes.md:22]: Change the content as we discussed in the teams call

File: MAUI/Themes/HowTo.md (hunk 2 above)
Comment (AarthiArjunan) [reply MAUI/Themes/HowTo.md:11]: Changes addressed.

File: MAUI/Themes/HowTo.md (hunk 3 above)
Comment (AarthiArjunan) [reply MAUI/Themes/HowTo.md:24]: Changes addressed.

File: MAUI/Themes/Themes.md (hunk 4 above)
Comment (AarthiArjunan) [reply MAUI/Themes/Themes.md:22]: Changes addressed.

File: MAUI/Themes/Themes.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+<Application xmlns:base="clr-namespace:SampleBrowser.Maui.Base;assembly=SampleBrowser.Maui.Base"
+             xmlns:syncTheme="clr-namespace:Syncfusion.Maui.Themes;assembly=Syncfusion.Maui.Core"
+             x:Class="SampleBrowser.Maui.App"
+             ...>
+<Application.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <!-- Theme resource dictionary -->
+                <syncTheme:SyncfusionThemeResourceDictionary SfVisualTheme="LightDefault"/>
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Themes/Themes.md:45]: change the theme name

File: MAUI/Themes/Themes.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+In addition to the primary keys, the theme resource dictionary also contains the keys that are specific to each controls; these keys can also be overridden. You can find the keys and the UI elements to which they are mapped to all the controls in this `documentation`.
+
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.Maui.Themes;assembly=Syncfusion.Maui.Core"
+             ...>
+<Application.Resources>
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+            <syncTheme:SyncfusionThemeResourceDictionary SfVisualTheme="DarkDefault"/>
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Themes/Themes.md:75]: Change the theme name in whole PR

File: MAUI/Themes/Themes.md (hunk 5 above)
Comment (PaulAndersonS) [inline MAUI/Themes/Themes.md:45]: Theme name changed now

Comment (AarthiArjunan) [reply MAUI/Themes/Themes.md:45]: Changes addressed.

File: MAUI/Themes/Themes.md (hunk 6 above)
Comment (AarthiArjunan) [reply MAUI/Themes/Themes.md:75]: Changes addressed.

File: MAUI/Themes/Themes.md (hunk 5 above)
Comment (AarthiArjunan) [reply MAUI/Themes/Themes.md:45]: Changes addressed.

File: MAUI/Themes/Keys.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, you can customize the appearance of SfAvatarView without merging common theme resource and control style resource dictionaries.
+			
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.Maui.Themes;assembly=Syncfusion.Maui.Core"
+             ...>
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Themes/Keys.md:49]: Please check and remove the greater than symbol

**Review by PaulAndersonS — APPROVED** [review]

**Pull request merged**